R/testthat.R

```diff
@@ -142,20 +206,12 @@ testthat_database <- function(datasource, label = NULL, tests = "default") {
   }
 
   # dplyr test orchestrator
-  verbs <- c(
-    "mutate",
-    "filter",
-    "summarise",
-    "group_by",
-    "arrange"
-  )
-
   tests %>%
     map(~{
       curr_test <- .x
       context(names(curr_test))
       curr_test %>%
-        purrr::flatten() %>%
```

Comment:
Sometimes I have to explicitly pass the package because `rlang` and `purrr` both have a `flatten()` command, not sure if you ran into any issues during CMD check or at other times because of this.

Reply:
I did.  I will remove that, though, as I fixed the NAMESPACE so that only necessary functions are pulled in from `rlang`.